ddl/placement/const.go

```diff
@@ -28,3 +28,7 @@ const (
 	// RuleIndexIndex is the index for a rule of index.
 	RuleIndexIndex
 )
+
+// DCLabelKey indicates the key of label which represents the dc for Store.
+// FIXME: currently we assumes "dc" is the dcLabel key in Store
+const DCLabelKey = "dc"
```

Comment:
How about use `"zone"`, see https://github.com/tikv/tikv/blob/master/docs/how-to/deploy/location-awareness.md.

Reply:
updated.